tests/cypress/e2e/unit_tests/upgrade.spec.ts

```diff
@@ -50,21 +50,30 @@ describe('Upgrade tests', () => {
       if ( elementalUIVersion == '1.0.0' ) {
         cy.exec(`sed -i 's/# namespace: fleet-default/namespace: fleet-default/g' assets/managedOSVersionChannel.yaml`);
         cy.exec(`sed -i 's@image: %UPGRADE_CHANNEL_LIST%@image: ${upgradeChannelList}@g' assets/managedOSVersionChannel.yaml`);
-        cy.get('.nav').contains('Advanced').click();
-        cy.get('.nav').contains('OS Version Channels').click();
+        cy.get('.nav')
+          .contains('Advanced')
+          .click();
+        cy.get('.nav')
+          .contains('OS Version Channels')
+          .click();
         cy.clickButton('Create from YAML');
         // Wait needed to avoid crash with the upload
         cy.wait(2000);
-        cy.get('input[type="file"]').attachFile({filePath: '../../assets/managedOSVersionChannel.yaml'});
+        cy.get('input[type="file"]')
+          .attachFile({filePath: '../../assets/managedOSVersionChannel.yaml'});
```

Comment:
Again for later maybe: use var/const for the filename?

Reply:
same as above, code will be removed very soon